message_tagging_service/tagging_service.py

```diff
@@ -337,9 +337,12 @@ def tag_build(nvr, dest_tags, koji_session):
     :param str nvr: build NVR.
     :param dest_tags: tag names.
     :type dest_tags: list[str]
-    :return: pair of tag names, which are tagged to build successfully, and the
-        corresponding task id returned from Koji.
-    :rtype: list[(str, int)]
+    :return: a list of 3-tuple tag build info. The first element is tag name to
+        apply, the second one is the task id return from Koji, and the last one
+        is the error message. If a build is tagged successfully, error message
+        is set to None, otherwise None is set to task id and error message has
+        some content.
+    :rtype: list[(str, int, str)]
```

Comment:
Consider returning a list of [namedtuples](https://docs.python.org/3/library/collections.html#collections.namedtuple).

Reply:
Done